onnx/test/reference_evaluator_test.py

```diff
@@ -5339,6 +5340,163 @@ def test_regex_invalid_pattern(self):
         with self.assertRaises(ValueError):
             ref.run(None, {"X": np.array(["x"])})
 
+    def test_a_function_calling_a_function_once(self):
+        X = make_tensor_value_info("X", TensorProto.FLOAT, ["N"])
+        out = make_tensor_value_info("output", TensorProto.FLOAT, ["N"])
+        Z = make_tensor_value_info("output", TensorProto.FLOAT, ["N"])
```

Comment:
nit: using same name (eg., `output = make_tensor_value_info("output", TensorProto.FLOAT, ["N"])`) would be helpful in the above two lines.

Reply:
done